tests/test_slack_integration.py

```
@@ -0,0 +1,79 @@
+            "*Source:* `test cluster`"
+        ]
+    ),
+    (
+        SlackTemplateStyle.DEFAULT,
+        "custom_template.j2",
+        [
+            "CUSTOM TEMPLATE:",
+        ]
+    )
+])
+def test_slack_template_styles(slack_channel: SlackChannel, template_style, template_name, expected_phrases):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_slack_template_styles`

Add return type annotation: `None`

(ANN201)